app/components/ilios-users.hbs

```diff
@@ -65,14 +65,15 @@
       {{/if}}
     </section>
     {{#if this.searchForUsers.lastSuccessful}}
-      <PagedlistControls
-        @offset={{@offset}}
-        @limit={{@limit}}
-        @limitless={{true}}
-        @setOffset={{@setOffset}}
-        @setLimit={{@setLimit}}
-        data-test-top-paged-list-controls
-      />
+      <div data-test-top-paged-list-controls>
```

Comment:
I don't know why, but I'm sort of bothered by doing a div wrap just for testing. I don't see an easy way to target the element even if we pass the `data-test-` into the component though so let's go with it, mainly it's just late and I'm ornery about all code today.

Reply:
i know, i was annoyed by that too.